src/user/user.controller.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  Controller,
+  Req,
+  Get,
+  UseGuards,
+  UseInterceptors,
+  ClassSerializerInterceptor,
+} from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+
+import { Request } from 'express';
+
+import { UserService } from './user.service';
+import { GetMeOutput } from './dto/me.dto';
+
+@Controller('users')
+export class UserController {
+  constructor(private readonly userService: UserService) {}
+
+  @UseGuards(AuthGuard())
```

Comment:
Another small question, do we have any preference over using such implicit default guards over explicit ones? `AuthGuard('jwt')`
@war1oc @asad-mlbd @Saad-Amjad

Reply:
Conclusion: Think about something like `@UseGuards(JWTGuard)`

Reply:
Proceed with current implementation for now. Introduce github issues and future PRs for the modification.